src/constants/screen.ts

```diff
@@ -10,8 +10,10 @@ export const taskConstants: ScreenConstantsInterface = {
   'Check on-chain balance': "balance",
   'Get network info': "network",
   'Get validator info': "validators",
-  'Move assets from C-chain to P-chain': 'CP',
-  'Move assets from P-chain to C-chain': "PC",
+  'Export assets from C-chain to P-chain': 'CP',
```

Comment:
be consistent either use single quotes or double across the repo

Reply:
How do you identify the difference between export and import where you use pc and pc for both import and export.